cvise/utils/testing.py

```diff
@@ -294,15 +283,15 @@ def report_pass_bug(self, test_env, problem):
         if crash_dir is None:
             return False
 
-        os.mkdir(crash_dir)
+        crash_dir.mkdir()
         test_env.dump(crash_dir)
 
         if not self.die_on_pass_bug:
             logging.debug(
                 f'Please consider tarring up {crash_dir} and creating an issue at https://github.com/marxin/cvise/issues and we will try to fix the bug.'
             )
 
-        with open(os.path.join(crash_dir, 'PASS_BUG_INFO.TXT'), mode='w') as info_file:
+        with (crash_dir / 'PASS_BUG_INFO.TXT').open(mode='w') as info_file:
```

Comment:
This confused me at first.  The `pathlib.Path` class implements the `/` operator.

That's cute.

Reply:
Yep, it's neat!